docs-v2/integrations/all/netsuite.mdx

```diff
@@ -41,5 +41,53 @@ For more details see the [docs here](/integrate/guides/authorize-an-api#apis-req
+To avoid this we recommend using the machine to machine flow which is called the `netsuite-tba` integration in Nango
 
 <Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/netsuite.mdx)</Note>
+
+## Admin Setup for `netsuite-tba`
+-   Ensure you have admin privileges and login to your Netsuite account and navigate to **Setup** > **Company** > **Enable Features**
+-   Under the "**SuiteTalk**" header make sure the _SOAP WEB SERVICES_ and _REST WEB SERVICES_ checkboxes are checked
+<Frame>
+  <img src="/images/netsuite-soap-rest-checkboxes.png" />
+</Frame>
+-   Under the "**Manage Authentication**" header make sure the _TOKEN-BASED AUTHENTICATION_ checkbox is checked
+<Frame>
+  <img src="/images/netsuite-tba-checkbox.png" />
+</Frame>
+-   Click the save button
+-   Navigate to **Setup** > **Integration** > **Manage Integrations** > **New**
+-   Set the desired name of this integration
+-   Enable _TOKEN-BASED AUTHENTICATION_ and disable _TBA: AUTHORIZATION FLOW_ and _AUTHORIZATION CODE GRANT_
+<Frame>
+  <img src="/images/netsuite-tba-configuration.png" />
+</Frame>
+-   Click the save button
+-   The client credentials will now be displayed and be sure to copy the **Consumer Key**, **Client ID**, **Consumer Secret**, and **Client Secret**
+-   Navigate to the homepage by clicking the home icon
+-   At the button left corner click the **Manage Access Tokens** button
+<Frame>
+  <img src="/images/netsuite-create-token.png" />
+</Frame>
+-   Select the **Application Name** you created for this integration
+-   Enter a "**Token Name**"
+-   Save your new access token
+-   The token credentials will now be displayed. Copy the "**Token ID**" & "**Token Secret**"
+-   In Nango create a new integration using the `netsuite-tba` and leave the "Client Id" and "Client Secret" values empty.
+-   Create a new connection and for the "**OAuth Client Id Override**" value paste in the
+value you received from Netsuite called "**Client ID**" and for the "**OAuth Client Secret**" value paste in the value you received
+from Netsuite called "**Client Secret**". For "**Token ID**" & "**Token Secret**" paste in the values you received from Netsuite
+from the access token step. The account ID value can be found from your [Netsuite URL instance](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_1498754928.html)
+-   If done programmatically using the Nango frontend sdk:
+```js
+nango.auth('netsuite-tba', connectionId, {
+  params: { accountId: values.accountId },
+  credentials: {
+    token_id: values.tokenId,
+    token_secret: values.tokenSecret,
+    oauth_client_id_override: values.clientId,
+    oauth_client_secret_override: values.clientSecret,
```

Comment:
I would omit the `values.` prefix. I don't feel like it is bringing much.

```
nango.auth('netsuite-tba', connectionId, {
  params: { accountId: ACCOUNT_ID},
  credentials: {
    token_id: TOKEN_ID,
    token_secret: TOKEN_SECRET,
    oauth_client_id_override: CLIENT_ID,
    oauth_client_secret_override: CLIENT_SECRET,
```

Reply:
True, updated with https://github.com/NangoHQ/nango/pull/2743/commits/90ae93802e3cddb303e2b113446332a0211c34f3